rust_src/src/windows.rs

```diff
@@ -1074,6 +1074,35 @@ pub fn window_new_total(window: LispWindowValidOrSelected) -> LispObject {
     win.new_total
 }
 
+/// Set new total size of WINDOW to SIZE.
+/// WINDOW must be a valid window and defaults to the selected one.
+/// Return SIZE.
+///
+/// Optional argument ADD non-nil means add SIZE to the new total size of
+/// WINDOW and return the sum.
+///
+/// The new total size of WINDOW, if valid, will be shortly installed as
+/// WINDOW's total height (see `window-total-height') or total width (see
+/// `window-total-width').
+///
+/// Note: This function does not operate on any child windows of WINDOW.
+#[lisp_fn(min = "2")]
+pub fn set_window_new_total(
+    window: LispWindowValidOrSelected,
+    size: EmacsInt,
+    add: LispObject,
```

Comment:
It would be more in line with conventions to make `add` an `Option<LispObject>`  and then `match` on it, similarly to how the example in `README.md` does.

Reply:
👍

Reply:
I have a question!
Which the ADD would be if LispObject is nil, Some(Qnil) or None?

Reply:
That'd be None

Reply:
We tend to skip `Option<LispObject>`. It does not add much _and_ you still need to check if the object is nil.

Reply:
You want `add` to just be a `bool` in this case :)

Reply:
👍